django/api/services/bcngws.py

```diff
@@ -0,0 +1,52 @@
+import requests
+from django.conf import settings
+
+def get_placename_matches(names_list):
+    current_index = 1
+    total_results = 200 #temporary
+    names_string = ''
+    names_string = ", ".join([str(item) for item in names_list])
```

Comment:
I think this could just be initialized 1 time here instead of starting it as empty. Other than that LGTM!